proposals/PROPOSALS.md

```diff
@@ -1,141 +1,125 @@
+
+The current members of the TWG are:
+ * Davean Scies (@davean)
+ * Gershom Bazerman (@gbaz)
+ * John Ericson (@Ericson2314)
+ * Hécate (@Kleidukos)
+ * Evie Ciobanu (@eviefp)
+ * David Thrane Christiansen (@david-christiansen)
+
+## Decisions
+
+The committee will make all decisions using a simple majority without secret ballots.
+Discussions and vote tallies regarding potential committee members are to be kept strictly confidential, while other discussions and votes should be transparent to the public.
+Decisions should be made with the participation of 2/3 of the current members of the committee, rounded up.
+The committee may decide to hold regular meetings, or to collaborate asynchronously, or some mix of the two.
+For instance, a valid decision can be made by having a non-quorum meeting discuss a matter, summarize the results for the other members, and then vote by email based on the summary.
+Disputes will be resolved by the ED.
+
+## Process
+
+The TWG, as an advisory committee to the Haskell Foundation and to the community at large, exists primarily to aid in making good decisions. 
+
+All proposals should address the following concerns:
+ * **Problem statement:** What problem is the proposed work intended to solve? What are the requirements against which a solution should be evaluated? How will solving this problem benefit the community?
+ * **Prior art:** What other similar work has been done in the community, and how is the proposal related to it?
+ * **Related efforts:** What other related activities are planned or ongoing? How is the proposal connected to them?
+ * **Technical and organizational work**: What work is actually being proposed? What technology should be developed, and how will it be managed and cared for going forward?
+ * **Success:** What does it mean for the project to have succeeded?
+ * **Stakeholders:** Who will be affected, and how?
+ * **Time:** How long will the proposal take to implement?
+
+Additionally, projects seeking HF support or execution should additionally address:
+ * **Budget:** How much time and money will the project cost initially? What about ongoing maintenance?
+ * **Additional partners:** Who else will participate in the project?
+
+One way to address these concerns is to begin with one of the [proposal templates](templates/).
+
+
+### Pre-Proposal
+Prior to submitting a detailed proposal, please post to the Haskell Discourse instance with a summary of the proposal's contents and a subject line that begins with "Pre-HFTP:".
+This helps address the prior art and related efforts concerns, and can save time needed to rewrite a proposal in light of previously-unknown opportunities for collaboration.
+While submitters are strongly encouraged to discuss their proposal on Discourse ahead of time, proposals will not be summarily rejected for not having done so.
+It is sufficient to post a link to the final discussion thread and invite comments and discussion.
```

Comment:
I do hope that people will post vague ideas to Discourse before attempting to write most proposals in detail, as once a detailed text has been written, it becomes more difficult to adapt to new knowledge that might arise.

That said, the fundamental concern being addressed here is that proposal authors are aware of what else has been done, and that they are aware of others working on related projects, and that the proposal takes this into account.

Reply:
This sentence is now:
>It is sufficient to post a link to an existing draft of the proposal to solicit prior art and related work before revising and formally submitting the proposal.
Does this address your concern?

Reply:
If you mean

* It is fine to initiate the Discourse discussion based on a vague or as-yet unformed idea; but it is *also* fine to create a proposal in (a fork of) the repo, and post that link to Discourse, as a way to solicit prior art and related work.  This would naturally occur between step 3 and 4 of the process below.

Then yes.

Reply:
The step-by-step process here is perhaps too explicit. We also won't require that people make a new fork of the repo every time they submit a proposal, after all. The important thing is that we avoid stepping on toes, and that the proposer takes new knowledge of related efforts seriously.

I kept the step-by-step instructions from the old version of the procedure, but I think I'll replace them with a description of the desired outcome, which is a PR with a certain kind of file in it where some background work has been done. People can arrive at that situation however they want, and I think that facility with Github is reasonable to assume amongst those who might submit proposals.